src/OrchardCore.Modules/OrchardCore.Media.Azure/Startup.cs

```diff
@@ -15,9 +15,9 @@ namespace OrchardCore.Media.Azure
     public class Startup : StartupBase
     {
         private ILogger<Startup> _logger;
-        private readonly IShellConfiguration _configuration;
+        private readonly ITenantConfiguration _configuration;
```

Comment:
If you start renaming Shells into Tenants, you will get @PinpointTownes angry

Reply:
Ah okay, no problem i will revert it back to `IShellConfiguration`.

Reply:
@sebastienros as long as it's consistent, I have no problem with that being called "shell" or "tenant" :smile:

Reply:
Too late, already reverted 😄

Reply:
While you're here, would that be worth renaming `TenantState` to `ShellState`? :trollface:

Reply:
Just for infos `ShellState` is an existing class ;)

    public class ShellState
    {
        public List<ShellFeatureState> Features { get; } = new List<ShellFeatureState>();
    }

More seriously @sebastienros @PinpointTownes for naming it is as you want, just let me know.

Reply:
I would need to resurrect an old chat with Kevin where he explained me the discrepancies and how to fix it. In the meantime no changes.